### DISCUSSION TITLE
does slint(rust) support UI modularity plugin, like bevy，personal define or create new modularity for slint

### DISCUSSION THREAD

**Discussion by banrieen** [slint-ui/slint#6271] [Ideas]
Existing GUI frameworks typically support communities or third-party design and development of a variety of UI components, themes, and animations, which can make development more efficient and adaptable to a wider range of scenarios.

**Comment by ogoffart** [comment]
There are some repository with third party components.
For example https://app.radicle.xyz/nodes/seed.radicle.garden/rad:z3oxAZSLcyXgpa7fcvgtueF49jHpH and https://github.com/Surrealism-All/SurrealismUI and https://github.com/slint-ui/slint/discussions/6036 and there might be more

**Reply by banrieen** [reply]
Thanks a lot, that are very usefull, and I Hope there are more components like this that can be easily referenced.

**Comment by banrieen** [comment]
Thasks a lot
